tests/QueryBuilderTest.php

```diff
@@ -574,6 +574,33 @@ public function testDates()
         $this->assertCount(2, $users);
     }
 
+    public function testImmutableDates()
+    {
+        DB::collection('users')->insert([
+            ['name' => 'John Doe', 'birthday' => new UTCDateTime(Date::parse("1980-01-01 00:00:00")->format('Uv'))],
+            ['name' => 'Robert Roe', 'birthday' => new UTCDateTime(Date::parse("1982-01-01 00:00:00")->format('Uv'))],
+        ]);
+
+        $user = DB::collection('users')->where('birthday', '=', new DateTimeImmutable("1980-01-01 00:00:00"))->first();
+        $this->assertNotNull($user);
```

Comment:
Add `assertCount` for check count elements

Reply:
Changed as requested ✅